### PR TITLE
clang stuff

### PR DESCRIPTION
**Pull request by neheb**

### CODE REVIEW

File: src/storage/sql_storage.cc [hunk 1]
```diff
@@ -481,17 +481,14 @@ std::vector<std::shared_ptr<CdsObject>> SQLStorage::browse(const std::unique_ptr
     std::shared_ptr<SQLResult> res;
     std::unique_ptr<SQLRow> row;
 
-    bool haveObjectType = false;
-
-    if (!haveObjectType) {
+    {
```
Comment (whyman) [inline src/storage/sql_storage.cc:484]: Shouldn't this block be removed instead?

File: src/transcoding/transcoding.cc [hunk 2]
```diff
@@ -67,7 +67,6 @@ TranscodingProfile::TranscodingProfile(transcoding_type_t tr_type, std::string n
     buffer_size = 0;
     chunk_size = 0;
     initial_fill_size = 0;
-    tr_type = TR_None;
```
Comment (whyman) [inline src/transcoding/transcoding.cc:70]: Looks fine?

File: src/storage/sql_storage.cc (hunk 1 above)
Comment (neheb) [reply src/storage/sql_storage.cc:484]: I tried that. There's some variable redefinition stuff going on. Easiest to just keep it.

File: src/transcoding/transcoding.cc (hunk 2 above)
Comment (neheb) [reply src/transcoding/transcoding.cc:70]: ¯\_(ツ)_/¯

File: src/storage/sql_storage.cc (hunk 1 above)
Comment (whyman) [reply src/storage/sql_storage.cc:484]: That makes it more worrying!

**Comment by whyman** [comment]
Other than my two comments the rest looks fine

File: src/transcoding/transcoding.cc (hunk 2 above)
Comment (neheb) [reply src/transcoding/transcoding.cc:70]: The issue pointed out by clang-analyzer is that it's a dead store. It's never read.

File: src/storage/sql_storage.cc (hunk 1 above)
Comment (neheb) [reply src/storage/sql_storage.cc]: Removed. Fixed the variable redefinition later on by clearing it before use.

**Comment by neheb** [comment]
Fine now?

**Pull request merged**